WBooks/Screens/BookLibrary/BookLibraryController.swift

```diff
@@ -44,13 +48,21 @@ class BookLibraryController: UIViewController {
         navigationItem.leftBarButtonItems = [notifications]
     }
     
+    private func initLibraryTableViewModel() {
+        libraryViewModel.reloadViewClosure = { [weak self] () in
```

Comment:
This reload closure is executed when the books are loaded. You could send this as a parameter for `loadBooks`